src/azure/azure-mesh-adapter.ts

```diff
@@ -24,21 +25,68 @@ export class AzureMeshAdapter implements MeshAdapter {
     tenants: MeshTenant[],
     startDate: Date,
     endDate: Date,
+  ): Promise<void> {
+    const config = loadConfig();
+    if (config.azure.parentManagementGroups.length == 0) {
+      log.warning(
+        "Please configure an Azure management group ID under which your Subscriptions are placed. " +
+          "Because of a bug in the Azure API collie can not detect this automatically and with this info cost usage lookups are significantly faster.",
```

Comment:
When mentioning `Collie` use one of the static variables in the config.

Reply:
done